.github/workflows/wait-for-build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Wait for build
         uses: lewagon/wait-on-check-action@v1.2.0
         with:
-          ref: ${{ github.ref }}
+          ref: ${{ github.sha }}
           check-name: 'Building PKI'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           wait-interval: 30
```

Comment:
Do we need to change this to `60` as well?

Reply:
Probably a good idea, will do :-)